tests/foreman/ui/test_repositories.py

```diff
@@ -0,0 +1,60 @@
+"""Test module for Repositories UI.
+
+:Requirement: Repository
+
+:CaseAutomation: Automated
+
+:CaseLevel: Component
+
+:CaseComponent: Repositories
+
+:team: Phoenix-content
+
+:TestType: Functional
+
+:CaseImportance: Critical
+
+:Upstream: No
+"""
+import pytest
+
+
+@pytest.mark.rhel_ver_match('[^6]')
+def test_positive_custom_products_disabled_by_default(
```

Comment:
I think this test is repeating the steps we are asserting in https://github.com/SatelliteQE/robottelo/pull/11610/files#diff-f463019213cc308428eb260208f12e2900768968b235902882346a2344e74e6eR23

Should we have this test at all which is already being asserted in different test ?

Reply:
two different tests, here we are testing that products are disabled by default. And in the other test, we are testing the "select all" feature for custom repos